lib/pages/chat/events/message_content.dart

```diff
@@ -236,9 +236,6 @@ class MessageContent extends StatelessWidget with PlayVideoActionMixin {
               );
             }
 
-            final bigEmotes = event.onlyEmotes &&
```

Comment:
why do you remove it

Reply:
it's for when amount of emojis is bigger than 10, the emojis will be small.

Reply:
no need at the moment